### PR TITLE
IDEMPIERE-5180 : Add client name on RoleAccessUpdate process and add …

### PR DESCRIPTION
**Pull request by nmicoud**
…client validation

### CODE REVIEW

**Review by hengsin — CHANGES_REQUESTED** [review]
please see comments at ticket.

File: org.adempiere.base.process/src/org/compiere/process/RoleAccessUpdate.java [hunk 1]
```diff
@@ -121,7 +126,7 @@ protected String doIt () throws Exception
 	 */
 	private void updateRole (MRole role)
 	{
-		StringBuilder msglog = new StringBuilder(role.getName()).append(": ") 
+		StringBuilder msglog = new StringBuilder(getAD_Client_ID() > 0 ? "" : MClient.get(role.getAD_Client_ID()).getName() + " - ").append(role.getName()).append(": ")
```
Comment (hengsin) [inline org.adempiere.base.process/src/org/compiere/process/RoleAccessUpdate.java:129]: I think it is fine to remove the ad_client_id > 0 check and always show the tenant/client name.

Comment (nmicoud) [reply org.adempiere.base.process/src/org/compiere/process/RoleAccessUpdate.java]: done

**Comment by hengsin** [comment]
Hi @nmicoud , the changes looks good but you need to resolve the merge conflict.

**Comment by nmicoud** [comment]
hi @hengsin , I think is done (is my first merge conflict resolution)

**Review by hengsin — APPROVED** [review]
looks good.

**Pull request merged**